src/__tests__/App.test.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from "@testing-library/react";
+import App from "../components/App";
+
+describe('App', () => {
+  const { asFragment } = render(<App />)
+
+  test("app renders correctly", () => {
```

Comment:
Normally you would render the component inside each test block, could easily do this here:
```
  test("app renders correctly", () => {
      const { asFragment } = render(<App />)
```